### PR TITLE
RISCV: Tidy fixes for RISCV class

### PR DESCRIPTION
**Pull request by dlrobertson**
## RISCV: Tidy fixes for RISCV class

Fixes for comment in #310 

### How Has This Been Tested? ###

| Architecture | Yes/No                   | Comments               |
|--------------|:------------------------:|------------------------|
| x86-32       | :heavy_multiplication_x: |   |
| x86-64       | :heavy_multiplication_x: |                        |
| ARM          | :heavy_multiplication_x: |                        |
| AARCH64      | :heavy_multiplication_x: |                        |
| MIPS         | :heavy_multiplication_x: |                        |
| POWERPC      | :heavy_multiplication_x: |                        |
| SPARC        | :heavy_multiplication_x: |                        |

Basic testing with RISC-V

### Checklist ###

- [x] My code follows the code style of this project.
- [x] My change includes a change to the documentation, if required.
- [x] I have read and agree to the **CONTRIBUTING** document.

### CODE REVIEW

File: gef.py [hunk 1]
```diff
@@ -1463,8 +1463,8 @@ def convert_to_signed(x):
         # If the conditional operation is not unsigned, convert the python long into
         # its two's compliment
         if not condition.endswith("u"):
-            rs2 = convert_to_signed(rs2)
-            rs1 = convert_to_signed(rs1)
+            rs2 = long_to_twos_compliment(rs2)
```
Comment (Grazfather) [inline gef.py]: Haha it's complement, not compliment.

File: gef.py [hunk 2]
```diff
@@ -1463,8 +1463,8 @@ def convert_to_signed(x):
         # If the conditional operation is not unsigned, convert the python long into
         # its two's compliment
```
Comment (Grazfather) [inline gef.py]: complement, not compliment.

File: gef.py [hunk 3]
```diff
@@ -1433,17 +1433,17 @@ def is_conditional_branch(self, insn):
         return insn.mnemonic.startswith("b")
 
     def is_branch_taken(self, insn):
-        def convert_to_signed(x):
+        def long_to_twos_compliment(v):
             """Convert a python long value to its two's compliment"""
             if is_elf32():
-                if (x & 0x80000000) != 0:
-                    return x - 0x100000000
+                if v & 0x80000000 != 0:
```
Comment (Grazfather) [inline gef.py]: you don't need the != 0, just `if v & ...`

File: gef.py (hunk 2 above)
Comment (dlrobertson) [reply gef.py]: :man_facepalming:

File: gef.py (hunk 1 above)
Comment (dlrobertson) [reply gef.py]: :man_facepalming:

File: gef.py (hunk 3 above)
Comment (dlrobertson) [reply gef.py]: True

**Pull request merged**